src/roadmap/release-checklist.md

```diff
@@ -34,43 +34,59 @@ A release consists of the following artifacts:
 images referenced above. For example, if `$WORKFLOW_RELEASE` is `2.0.0-beta3`, the new chart would
 be in a new directory called `workflow-beta3`.
 
-# Step 1: Get the Status of all Components
+# Step 1: Create New Helm Classic Charts
 
-First, we'll need to get the statuses of all repositories that house the components we're
-interested in upgrading. We'll use
-[sgoings/deis-workflow-group](https://github.com/sgoings/deis-workflow-group) to do that in one
-place. That repository is a group of git submodules with all of the applicable repositories in it,
-so that we can manage everything from one place.
-
-Clone that repository to any location on your local machine, update all submodules and list
-the latest commit for each submodule:
-
-```console
-git clone https://github.com/sgoings/deis-workflow-group
-cd deis-workflow-group
-make git-update
-git submodule status
+First, export necessary values for `WORKFLOW_RELEASE` and `WORKFLOW_RELEASE_SHORT`:
+```
+export WORKFLOW_RELEASE=<full release name>
+export WORKFLOW_RELEASE_SHORT=<short form of above>
 ```
 
-Keep the list of commit SHAs handy - you'll need it for later.
-
-# Step 2: Create a New Helm Classic Chart
-
-Next, we'll create a new [Helm Classic](https://github.com/helm/helm-classic) chart so that we can "stage" a
-version of our release for testing. The process is fairly simple:
-
-1. Create a new branch: `git checkout -b release-$WORKFLOW_RELEASE`
-2. Copy the existing `dev` chart: `cp -r workflow-dev workflow-$WORKFLOW_RELEASE_SHORT`
-3. Modify the `workflow-$WORKFLOW_RELEASE_SHORT/tpl/generate_params.toml` file to ensure that all
-`dockerTag` values look like `git-$COMPONENT_SHA_SHORT`, where `$COMPONENT_SHA_SHORT` is the first
-7 characters of the applicable SHA that you got in the previous step.
-4. Ensure that all `DEBUG` variables in manifests are `false`.
-5. Delete the `KUBERNETES_POD_TERMINATION_GRACE_PERIOD_SECONDS` env var from `tpl/deis-controller-rc.yaml`
-6. Commit your changes: `git commit -a -m "chore(workflow-$WORKFLOW_RELEASE_SHORT): releasing workflow-$WORKFLOW_RELEASE_SHORT"`
-7. Push your changes to your fork: `git push -u $YOUR_FORK_REMOTE release-$DEIS_RELEASE`. Note that
-`$YOUR_FORK_REMOTE` is the git URI to the remote of your `deis/charts` fork. Mine is `git@github.com:arschles/deis-charts.git`, for example.
-8. Do steps 2-5 with the `workflow-beta3-e2e` directory
-9. Open a pull request from your branch to merge into `master` on https://github.com/deis/charts
+Next, we'll create new [Helm Classic](https://github.com/helm/helm-classic) charts so that we can "stage" a
+version of our release for testing. Here is the current process to do so:
+
+1. Create a new branch in [deis/charts](https://github.com/deis/charts): `git checkout -b release-$WORKFLOW_RELEASE`
```

Comment:
should this be `git checkout -b release-$WORKFLOW_RELEASE deis/master` for clarity?